_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
+     */
+    function renderTable(data, domain= false ) {
+        let table = ``;
+        let domainTables = ``;
+        let tableDomains = []; // list of domains
+        let tableDomainhtml = {}; // hold html table data with domain as key
+        for (let i = 0; i < data.length; i++) {
+            let id = data[i]['id'];
+            let is_obsolete = "";
+            let is_inactive = "";
+            let activity_status = data[i]['activity_status']
+            let title = data[i]['title'];
+            let license_url = "#";
+            let license_logo = "#";
+            let license_label = "";
+            let description = data[i]['description'];
+            let repo = "";
+            let homepage = data[i]["homepage"];
+            let tracker = "";
+            let contact = "";
+            let publication = "";
+            let foundry_order = "";
+            let domainInner = ["Unknown"];
+
+            if (data[i]['license']) {
+                license_url = data[i]["license"]["url"];
+                license_logo = data[i]["license"]["logo"];
+                license_label = data[i]["license"]["label"];
+            }
+            if (data[i]["repository"] && data[i]["repository"].includes("https://github.com/")) {
+                repo = data[i]["repository"];
+            }
+            if (data[i]["tracker"]) {
+                tracker =`<a class="btn btn-default btn-sm" href="${data[i]["tracker"]}" aria-label="Go to the tracker for ${title}" title="Go to the tracker for ${title}">
+                                <span class="glyphicon glyphicon-comment" aria-hidden="true"></span>
+                            </a>`;
+            }
+            if (data[i].hasOwnProperty("domain") && data[i]['domain'] !== undefined) {
+                domainInner[0] = data[i]['domain'];
+            }
+            if (description !== undefined && description.toString().length > 140) {
+                description = description.toString().slice(0, 140) + '...'
+            }
+            if (data[i]["contact"]) {
+                contact =`
+                        <a class="btn btn-default btn-sm" href="mailto:${data[i]["contact"]["email"]}" aria-label="Send an email to ${title}"
+                           title="Send an email to ${title}">\
+                            <span class="glyphicon glyphicon-envelope" aria-hidden="true"></span>
+                        </a>`;
+            }
+            if (data[i]["publications"] && data[i]["publications"].length > 0) {
+                publication = `
+                            <a class="btn btn-default btn-sm" href="${data[i]["publications"][0]["id"]}" aria-label="View the primary publication for ${title}" title="View the primary publication for ${title}">
+                                <span class="glyphicon glyphicon-book" aria-hidden="true"></span>
+                            </a>`;
+            }
+            if (data[i]["in_foundry_order"]) {
```

Comment:
Delete this (related to https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1978/files#r933982967)

Reply:
Yes I added this to the TODO list, thanks for finding!